e2-studio-star-rx72n-firmware/src/boot/r_bsp/r_rx_intrinsic_functions.c

```
@@ -0,0 +1,1237 @@
+ *
+ * @note GNUC-specific implementation (C fallback)
+ * @note Not thread-safe if addr1 or addr2 arrays are shared
+ * @note Typically used via R_BSP_RMPAB macro
+ *
+ * @see R_BSP_MulAndAccOperation_W() Word-sized variant
+ * @see R_BSP_MulAndAccOperation_L() Long-sized variant
+ * @see R_BSP_RMPAB Portable macro wrapper
+ * @since Version 1.0.0
+ */
+#if defined(__GNUC__)
+long long R_BSP_MulAndAccOperation_B(long long     init,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'R_BSP_MulAndAccOperation_B' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)